L1Trigger/CSCTriggerPrimitives/test/CSCTriggerPrimitivesReader.h

```diff
@@ -11,33 +11,125 @@
  *
  */
 
-#include "FWCore/Utilities/interface/EDGetToken.h"
-#include "FWCore/Framework/interface/Frameworkfwd.h"
-#include "FWCore/Framework/interface/EDAnalyzer.h"
-#include "FWCore/Framework/interface/Event.h"
-#include "FWCore/Framework/interface/EventSetup.h"
-#include "FWCore/ParameterSet/interface/ParameterSet.h"
-#include "FWCore/Utilities/interface/InputTag.h"
-
-#include "L1Trigger/CSCCommonTrigger/interface/CSCConstants.h"
-
-#include "DataFormats/CSCDigi/interface/CSCALCTDigiCollection.h"
-#include "DataFormats/CSCDigi/interface/CSCCLCTDigiCollection.h"
-#include "DataFormats/CSCDigi/interface/CSCCorrelatedLCTDigiCollection.h"
-#include "DataFormats/CSCDigi/interface/CSCWireDigiCollection.h"
-#include "DataFormats/CSCDigi/interface/CSCComparatorDigiCollection.h"
-
-#include "L1Trigger/CSCTriggerPrimitives/src/CSCCathodeLCTProcessor.h" // TMB07
-
-#include "SimDataFormats/TrackingHit/interface/PSimHitContainer.h"
-
-#include "TH1.h"
-#include "TH2.h"
+#include <FWCore/Utilities/interface/EDGetToken.h>
```

Comment:
These cmssw includes should rather stay within "". I do not see why they should be changed to <>.